lib/hammer_cli_katello/host_extensions.rb

```diff
@@ -57,18 +57,28 @@ module HostExtensions
                   _("Content View Environment Labels"), Fields::Field
             collection :content_view_environments, _('Content View Environments') do
               from :content_view do
-                label _("Content view") do
+                # Deprecated label. To be removed in future versions.
+                label _("Content view"), :sets => ['ALL'] do
```

Comment:
Shot in the dark, but is there any chance that a field definition somewhere around here could alleviate the issue?